packages/services/src/services/CloudAccountAuth/__tests__/CloudAccountAuth.test.js

```diff
@@ -14,7 +14,7 @@ describe('CloudAccountAuth cookie utility', () => {
       value: 'com.ibm.cloud.iam.LoggedIn.prod=1',
```

Comment:
probably need to update the mocking of the cookie in these two tests. Should be similar to the `DDO.test.js` tests

```
import root from 'window-or-global';

describe('CloudAccountAuth personalization utility', () => {
    beforeEach(function() {
        root._dl.ddo.user.segment.isCloudLoggedOn = true;
    });

 .....
 
}

Reply:
@annawen1 I've tried the suggestion that you've noted above, but it's failing. Maybe there's something that I'm missing?

```
import CloudAccountAuthAPI from '../CloudAccountAuth';
import root from 'window-or-global';

describe('CloudAccountAuth personalization utility', () => {
  it('should fetch the personalization window boolean and return the authenticated string', () => {
    root._dl.ddo.user.segment.isCloudLoggedOn = true;

    const loginStatus = CloudAccountAuthAPI.checkPersonalization();
    expect(loginStatus).toStrictEqual({ user: 'authenticated' });
  });

  it('should fetch the personalization window boolean and return the anonymous string', () => {
    root._dl.ddo.user.segment.isCloudLoggedOn = false;

    const loginStatus = CloudAccountAuthAPI.checkPersonalization();
    expect(loginStatus).toStrictEqual({ user: 'anonymous' });
  });
});
```

Reply:
Resolved by https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8149/commits/4a03d29ec62562d5a51f1539c42754dc5757b843